heed/src/envs/env.rs

```diff
@@ -441,6 +442,68 @@ impl<T> Env<T> {
         unsafe { self.copy_to_fd(fd, option) }
     }
 
+    /// Copy an LMDB environment to a file created at the given path, with options.
+    ///
+    /// This function may be used to make a backup of an existing environment.
+    /// No lockfile is created, since it gets recreated at need.
```

Comment:
What happens if there was already a lockfile? Shouldn’t we try to delete it?

Reply:
The lock file will not have any impact, we can just ignore it.